src/libradosstriper/RadosStriperImpl.cc

```diff
@@ -12,6 +12,8 @@
  *
  */
 
+#include <boost/algorithm/string/replace.hpp>
```

Comment:
@chardan And the top line of the commit message could be `libradosstriper: remove format injection vulnerability` . . . the entire commit message might then look like this:

```
libradosstriper: remove format injection vulnerability

Fixes: http://tracker.ceph.com/issues/20240
SIgned-off-by: Stan K <stan's email>
```